test/sink.test.js

```diff
@@ -43,7 +44,7 @@ test('.writer() - no value for "type" argument - should throw', () => {
     const sink = getValidSink();
     expect(() => {
         sink.writer();
-    }).toThrowError('"type" is missing');
+    }).toThrowError('Expected argument "type" to be provided to writer');
```

Comment:
Snapshot?

Reply:
Yea, thought about it. Figured since it was a short string i'd leave it but I could switch it.